compiler/ballerina-treegen/src/main/resources/syntax_tree_descriptor.json

```diff
@@ -2352,6 +2348,16 @@
             ]
         },
         {
+<<<<<<< HEAD
+            "name": "BindingPatternNode",
+            "base": "Node",
+            "kind": "BINDING_PATTERN",
+            "attributes": [
```

Comment:
Since there are different kinds of binding patterns, I think we need a structure like this:
```
Node
  |__BindingPatternNode
          |__CaptureBindingPatternNode
          |__ WildcardBindingPatternNode
          |__ ListBindingPatternNode
```